### ISSUE TITLE
AROMA not running in 23.1 

### ISSUE THREAD

**Issue by dmd** [nipreps/fmriprep#3045]
### What happened?

In 23.1.1, but not 23.0.2, AROMA seems to not run - but doesn't error out, it just behaves as if it hadn't been specified.

### What command did you use?

```shell
fMRIPrep version: 23.1.0

fMRIPrep command: /opt/conda/envs/fmriprep/bin/fmriprep /data/belleau_lab/TeenStress/E13347949 /data/belleau_lab/TeenStress/E13347949/derivatives participant --fs-license-file /cm/shared/freesurfer-license.txt --participant_label TS049 --output-spaces MNI152NLin6Asym:res-2 anat fsaverage --n_cpus 4 --mem-mb 16384 --notrack --dummy-scans 0 --use-aroma --use-syn-sdc --return-all-components -vv -w /data/fmriprep-workdir/ebelleau
```

### What version of fMRIPrep are you running?

23.1.0

### How are you running fMRIPrep?

Singularity

### Is your data BIDS valid?

Yes

### Are you reusing any previously computed results?

No

### Please copy and paste any relevant log output.

```shell
Here's the config it thinks it is starting with:

230614-14:08:46,994 nipype.workflow VERBOSE:
         fMRIPrep config:
                [environment]
                cpu_count = 32
                exec_env = "singularity"
                free_mem = 181.0
                overcommit_policy = "heuristic"
                overcommit_limit = "50%"
                nipype_version = "1.8.6"
                templateflow_version = "23.0.0"
                version = "23.1.1"

                [execution]
                bids_dir = "/data/belleau_lab/TeenStress/E13347949"
                bids_database_dir = "/data/fmriprep-workdir/ebelleau/20230614-140803_28b768a4-3108-471a-9419-6f55a68a6902/bids_db"
                bids_description_hash = "ecd58287c7e5111e4a4854c43e4d094e0bc8c59d52e1a549f4f5761fbb15bd7a"
                boilerplate_only = false
                sloppy = false
                debug = []
                fmriprep_dir = "/data/belleau_lab/TeenStress/E13347949/derivatives"
                fs_license_file = "/cm/shared/freesurfer-license.txt"
                fs_subjects_dir = "/data/belleau_lab/TeenStress/E13347949/derivatives/sourcedata/freesurfer"
                layout = "BIDS Layout: ...lleau_lab/TeenStress/E13347949 | Subjects: 1 | Sessions: 0 | Runs: 4"
                log_dir = "/data/belleau_lab/TeenStress/E13347949/derivatives/logs"
                log_level = 15
                low_mem = false
                md_only_boilerplate = false
                notrack = true
                track_carbon = false
                country_code = "CAN"
                output_dir = "/data/belleau_lab/TeenStress/E13347949/derivatives"
                me_output_echos = false
                output_layout = "bids"
                output_spaces = "MNI152NLin6Asym:res-2 anat fsaverage:den-164k"
                reports_only = false
                run_uuid = "20230614-140803_28b768a4-3108-471a-9419-6f55a68a6902"
                participant_label = [ "TS028",]
                templateflow_home = "/home/fmriprep/.cache/templateflow"
                work_dir = "/data/fmriprep-workdir/ebelleau"
                write_graph = false

                [workflow]
                anat_only = false
                aroma_err_on_warn = false
                aroma_melodic_dim = 0
                bold2t1w_dof = 6
                bold2t1w_init = "register"
                cifti_output = false
                dummy_scans = 0
                fmap_bspline = false
                force_syn = false
                hires = true
                ignore = []
                longitudinal = false
                medial_surface_nan = false
                project_goodvoxels = false
                regressors_all_comps = true
                regressors_dvars_th = 1.5
                regressors_fd_th = 0.5
                run_reconall = true
                skull_strip_fixed_seed = false
                skull_strip_template = "OASIS30ANTs"
                skull_strip_t1w = "force"
                slice_time_ref = 0.5
                spaces = "MNI152NLin6Asym:res-2 anat fsaverage:den-164k MNI152NLin2009cAsym"
                use_aroma = true
                use_syn_sdc = "error"
                me_t2s_fit_method = "curvefit"

                [nipype]
                crashfile_format = "txt"
                get_linked_libs = false
                memory_gb = 16.384
                nprocs = 4
                omp_nthreads = 3
                plugin = "MultiProc"
                resource_monitor = false
                stop_on_first_crash = false

                [seeds]
                master = 34084
                ants = 46217
                numpy = 28513

                [nipype.plugin_args]
                maxtasksperchild = 1
                raise_insufficient = false
```

### Additional information / screenshots

There are no errors in the html output. But when we run this with 23.0.2 with the exact same flags, we get AROMA output... when we run it with 23.1.1, we don't.

**Comment by dmd** [comment]
Aha! That would do it. Maybe don't accept the flag any more - or warn the user?

**Comment by effigies** [comment]
I think we do. Did it not show up?

https://github.com/nipreps/fmriprep/blob/e329aa1137775fdb4842945f554aec7e3cfacb2a/fmriprep/cli/run.py#L45-L56

**Comment by dmd** [comment]
It did not show up either in stdout or stderr:

```
root@micc /data/belleau_lab/TeenStress
$ grep -i aroma fmriprep.?185444
fmriprep.o185444:               aroma_err_on_warn = false
fmriprep.o185444:               aroma_melodic_dim = 0
fmriprep.o185444:               use_aroma = true

root@micc /data/belleau_lab/TeenStress
$
```

**Comment by effigies** [comment]
Ah. I should probably be using the logger, not regular warnings. Apologies. I need to put out a small patch in the next day or so, so I'll fix this for 23.1.2.

**Comment by effigies** [comment]
Should be resolved in 23.1.2.
